watsonsdk/AuthConfiguration.h

```diff
@@ -20,12 +20,14 @@
 
 @property NSString* basicAuthUsername;
 @property NSString* basicAuthPassword;
+@property (nonatomic) BOOL xWatsonLearningOptOut;
 
 @property (readonly) NSString *token;
 @property (copy, nonatomic) void (^tokenGenerator) (void (^tokenHandler)(NSString *token));
 
 - (void) invalidateToken;
 - (void) requestToken: (void(^)(AuthConfiguration *config)) completionHandler;
 - (NSDictionary*) createRequestHeaders;
+- (NSDictionary*) createRequestHeadersWithXWatsonLearningOptOut;
```

Comment:
I prefer to keep only one "createRequestHeaders" method because it is only SDK-internal use, and change value through config. If the developers want to change the SDK code, they could do it, we just keep the code simple. 

Thanks.